cairo-src/evm/utils.cairo

```diff
@@ -1,5 +1,6 @@
 from starkware.cairo.common.math import unsigned_div_rem
 from starkware.cairo.common.math_cmp import is_le
+from starkware.cairo.common.serialize import serialize_word
```

Comment:
Why are we importing serialize_word here? I'm pretty sure it doesn't exist on StarkNet

Reply:
That's for the time being, while we still aren't quite on starknet (although close). I need it to test that the output is correct.

Reply:
Ah yes, makes sense.